tests/manage/pv_services/pvc_snapshot/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import random
+import os
+import urllib.request
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import bugzilla
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources import pvc
+from ocs_ci.helpers import helpers
+from ocs_ci.framework.testlib import (
+    E2ETest,
+    tier1,
+    polarion_id,
+)
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
should we test this scenario for both blockpool and cephfs interfaces?

Reply:
Not sure about it